source/CIPlanetModel.cpp

```diff
@@ -54,6 +58,27 @@ void PlanetModel::dispose() {
     _planetNode = nullptr;
 }
 
+/**
+ * Initializes a new planet with the given color
+ *
+ * This method does NOT create a scene graph node for this planet.  You
+ * must call setTexture for that.
+ *
+ * @param x The initial x-coordinate of the center
+ * @param y The initial y-coordinate of the center
+ * @param c The initial color code of the planet
+ * @param maxLayers The maximum number of layers the planet can have
+ * @param gravStrength The planet's gravitational strength factor
+ * @param winCond The condition value for the planet to win
+ *
+ * @return true if the initialization was successful
+ */
+bool PlanetModel::init(float x, float y, CIColor::Value c, int maxLayers, float gravStrength, uint16_t winCond) {
```

Comment:
Is the other init method still used? If not we should get rid of it or put default values for maxLayers and gravStrength

Reply:
I think its being called in OpponentPlanet's alloc method.

Reply:
ok can you do the default values thing then. Take a look at addStardust in StardustQueue.h for an example